### ISSUE TITLE
Animate the evolution of repository in time

### ISSUE THREAD

**Issue by NikosAlexandris** [git-truck/git-truck#763]
How about being able to animate the evolution of the repository ? Like rolling slowly the end-time bar in the timeline axis and taking a snapshot every 'x' time step ? This would help to create a GIF or some animation.

**Comment by joglr** [comment]
Really neat idea. This is why we introduced the transitions, but currently you can only transition from one revision to another. Would be a cool feature :)

In the meantime, have you had a look at Gource? This will do exactly this, just with an animated graph of the repository instead of the hierarchical tree map and bubble chart we have in Git Truck.

**Comment by NikosAlexandris** [comment]
> Really neat idea. This is why we introduced the transitions, but currently you can only transition from one revision to another. Would be a cool feature :)
> 
> In the meantime, have you had a look at Gource? This will do exactly this, just with an animated graph of the repository instead of the hierarchical tree map and bubble chart we have in Git Truck.

Yes, I am playing already with Gource. Pure Awesomeness. One thing Gource is missing, seems to be, is to be a 3D-Model which you can rotate, spin and the likes except of pan and zoom which is there. Intend to write a Discussion about this in Gource's repository.

git-truck is also great. Wish the labels where more customisable - size at the very least and maybe the font too.

**Comment by joglr** [comment]
May I ask what you find particularly great about Git Truck? How does it fit into your workflow?

**Comment by NikosAlexandris** [comment]
> May I ask what you find particularly great about Git Truck? How does it fit into your workflow?

I need to _communicate_ the amount of work put in to a software. At least, try to help people relate, somehow, to it. Visualising its components is maybe one way. GitHub has https://githubnext.com/projects/repo-visualization/ (which looks similar to Git Truck ?). But I feel it's restrictive. I want to use such a visualiser with any project, not just and through GitHub.
